src/server/plugins/engine/models/FormModel.ts

```diff
@@ -49,7 +49,7 @@ export class FormModel {
 
   lists: FormDefinition['lists']
   sections: FormDefinition['sections'] = []
-  options: { basePath: string; defaultPageController?: string }
+  options: { basePath: string; defaultPageController?: string; formId: string }
```

Comment:
Resolved in 3eef78a5b92f4cf10c44df6427c480c9221263d2

It does but I'll make it optional in the types to save any pain in the test files.
There's always the chance we could want to use the runner in JSON file mode like the tests do (rather than from a form in the DB with metadata etc.)

```
await createServer({
  formFileName: 'titles.json',
  formFilePath: join(testDir, 'definitions')
})
```